prow/integ-suite-kind.sh

```diff
@@ -171,6 +171,16 @@ fi
 if [[ -z "${SKIP_BUILD:-}" ]]; then
   trace "setup kind registry" setup_kind_registry
   trace "build images" build_images "${PARAMS[*]}"
+  WASM_ATTRGEN_TAG="359dcd3a19f109c50e97517fe6b1e2676e870c4d"
+  docker pull gcr.io/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG
+  docker tag gcr.io/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG localhost:5000/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG
```

Comment:
nit: `crane cp gcr.io/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG localhost:5000/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG` - much faster and avoids dependency on `docker` specifically

Reply:
Done.